### DISCUSSION TITLE
Interest in SSBs? AKA Site-specific browser or or single-site browser

### DISCUSSION THREAD

**Discussion by markstos** [zen-browser/desktop#1195] [Ideas]
Long ago Mozilla had the Prism project to launch a single website as a desktop app. More recently there has been support for these site-specific browser instances (SSBs) hidden behind the "browser.ssb.enabled" preference.  This is something is that Chrome has enabled by default. 

An example would be saving Gmail or Google Calendar as an "app" that you can launch separately.

These can be great for websites that you keep open all day and think about as a "first class app" and not as a browser tab. 

When they are considered first-class applications, you can for example switch to them directly with an app switcher, while switching to a tab can be two steps (first switch to browser, then switch to tab). 

Some more history about this feature in Firefox:
https://duckduckgo.com/?q=Firefox+%22browser.ssb.enabled%22&t=ffab&ia=web

This feature is one reason I have been using Chrome-based browsers. Which is ironic as Mozilla's Prism may have been the first to introduce this!

**Comment by roiniti** [comment]
Would be nice to have it, i want to migrate to Zen from Microsoft edge and I use the web as app feature a lot, i need this on Zen

**Comment by ke7in-zz** [comment]
+1

**Comment by jessekalil** [comment]
+1
